app/controllers/application_controller.rb

```diff
@@ -1,2 +1,4 @@
 class ApplicationController < ActionController::Base
+    def index
```

Comment:
Why is this here?

Reply:
we accidentally added an index method to this one instead of products controller.
Forgot to delete the method. It's gone now